pipelines/h4c/post_processing/v6/lstbin/h4c_lstbincheck_forKaiFeng.toml

```diff
@@ -0,0 +1,55 @@
+[Options]
```

Comment:
This looks like it shouldn't be in this PR. Could you move it to a new branch?

Reply:
deleted!